feat: Ae coin details

janmichek opened the pull request:
<!--- Ensure the title of the Pull Request follows conventional commits syntax. If it resolves an issue, provide its ID in the scope section. -->

## Description
resolves #749 

## Demo

https://github.com/aeternity/aescan/assets/15363559/85a209a8-387e-4055-a69b-8ed9f130b625




## Checklist:
<!--- Put an `x` in all the boxes that apply. -->
<!--- If your change requires a documentation PR, please link it appropriately -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)

Liubov-crypto reviewed (changes requested):
I think it's worth to check if text is placed in green filed when copy it: 
https://github.com/aeternity/aescan/assets/69896204/9f2a4278-1c3c-4adc-b270-097bed6d0ca1

Also for short names of tokens (the same on prod): 
https://github.com/aeternity/aescan/assets/69896204/92198165-f7fe-4bc5-9246-7725fa223800

janmichek commented:
> I think it's worth to check if text is placed in green filed when copy it:

Fixed


> Also for short names of tokens (the same on prod) and aligning is different from prod:

The layout is good. That's the thing we changed lately

michele-franchi commented on src/components/AeCoinPanel.vue:
```diff
@@ -0,0 +1,129 @@
+  fetchMarketStats(),
+]))
+
+const priceChangeSign = computed(() => priceChange.value > 0 ? '+' : '')
```
```suggestion
const priceChangeSign = computed(() => priceChange.value > 0 ? '+' : '')
```

Shouldn't it be:
```
> 0: '+'
=0: ''
<0: '-'
```

janmichek replied:
right, fixed

michele-franchi commented on src/utils/hints/aeCoinHints.js:
```diff
@@ -0,0 +1,8 @@
+export const aeCoinHints = {
+  aeCoin: 'æternity native coin.',
+  tokenSymbol: 'Reserved token name.',
```
```suggestion
  tokenSymbol: 'Reserved coin name.',
```

michele-franchi commented on src/utils/hints/aeCoinHints.js:
```diff
@@ -0,0 +1,8 @@
+export const aeCoinHints = {
+  aeCoin: 'æternity native coin.',
+  tokenSymbol: 'Reserved token name.',
+  price: 'Price of the token in AE and USD.',
```
```suggestion
  price: 'Price of the coin in AE and USD.',
```

michele-franchi reviewed (changes requested):
I suggested some minor changes.
